pages/docs/migration/migrating-to-v3.md

```diff
@@ -0,0 +1,225 @@
+---
+title: "Migrating to v3"
+---
+Migration to any new version is always difficult, and AsyncAPI is no exception, but we want to provide as smooth a transition as possible, and this is where this document comes in. It shows the breaking changes between AsyncAPI v2 and v3 in an interactive manner as well as providing some guidance on why it happened.
+
+For a detailed read through about all the changes (non-breaking as well), please do read X before this, at it might give you some more context about the full list of features in v3.
+
+import {Asyncapi3Comparison, Asyncapi3ChannelComparison, Asyncapi3IdAndAddressComparison, Asyncapi3MetaComparison, Asyncapi3OperationComparison} from '../../../components/Asyncapi3Comparison'
+
+<Asyncapi3Comparison className="my-8" />
+
+### Operation, Channel and message decoupling
+
+This is by far the most intrusive breaking change in v3 that completely splits out how operations, channels and messages are related to each other.
+
+<Asyncapi3ChannelComparison className="my-8" />
+
+In v2, it was impossible to reuse channels, and impossible to have more then one operation per channel, i.e. operation variants.
```

Comment:
```suggestion
In v2, it was impossible to reuse channels, and impossible to have more than one operation per channel, i.e. operation variants.
```
- This typo was pointed out by @CynthiaPeter in #1909 Thank you for pointing it out.